indico/modules/events/papers/models/reviews.py

```diff
@@ -119,7 +119,7 @@ def locator(self):
 
     @return_ascii
     def __repr__(self):
-        return format_repr(self, 'id', 'revision', 'user_id', proposed_action=None)
+        return format_repr(self, 'id', 'type', 'revision', 'user_id', proposed_action=None)
```

Comment:
Unrelated, but I would not include revision but revision_id here. Triggering lazyloads in the repr can easily cause errors that make debugging things harder.